sdk/src/test/scala/com/horizen/fixtures/SidechainBlockInfoFixture.scala

```diff
@@ -31,11 +33,26 @@ trait SidechainBlockInfoFixture extends MainchainBlockReferenceFixture {
     getRandomModifiersSeq(count)
   }
 
+  def getMainchainBaseInfoFromReferences(references: Seq[MainchainBlockReference], initialCumulativeHash: Array[Byte]): Seq[MainchainHeaderBaseInfo] = {
+    val allRefsMainchainHeaderBaseInfo: ArrayBuffer[MainchainHeaderBaseInfo] = new ArrayBuffer[MainchainHeaderBaseInfo]()
+    var prevCumulativeHash = initialCumulativeHash
+    references.foreach(ref => {
```

Comment:
FYI: instead of using `.foreach` you may use `.map` so no need of `val allRefsMainchainHeaderBaseInfo` declaration.